common/network-common/src/main/java/org/apache/spark/network/util/NettyUtils.java

```diff
@@ -162,7 +162,6 @@ public static PooledByteBufAllocator createPooledByteBufAllocator(
       Math.min(PooledByteBufAllocator.defaultNumDirectArena(), allowDirectBufs ? numCores : 0),
       PooledByteBufAllocator.defaultPageSize(),
       PooledByteBufAllocator.defaultMaxOrder(),
-      allowCache ? PooledByteBufAllocator.defaultTinyCacheSize() : 0,
```

Comment:
Although there is a description about this, could you add the link to Netty site about this change?

Reply:
done ~

Reply:
Thanks!

Reply:
Could you re-trigger your GitHub Action?
- https://github.com/LuciferYang/spark/actions/runs/762178571

Reply:
ok

Reply:
There are some netty 4.1.50 dependencies in sbt test classpath, I'm trying to add some `ExcludeRule` to fix it now.